stations-injector/src/main/solution/workshop/stations/StationsInjector.java

```diff
@@ -88,13 +90,14 @@ private void inject(RoutingContext ctx) {
 
         Flowable<String> fileFlowable = rxReadGunzippedTextResource("cff-stop-2016-02-29__.jsonl.gz");
 
-        Flowable<Map.Entry<String, Stop>> pairFlowable =
-          fileFlowable.map(StationsInjector::toEntry);
+        Flowable<Map.Entry<String, Stop>> pairFlowable = fileFlowable.map(StationsInjector::toEntry);
 
-        Flowable<?> putFlowable =
-          pairFlowable.doOnNext(e -> stations.putAsync(e.getKey(), e.getValue()));
+        Completable completable = pairFlowable.map(e -> {
+          CompletableFuture<Stop> putCompletableFuture = stations.putAsync(e.getKey(), e.getValue());
+          return CompletableInterop.fromFuture(putCompletableFuture);
+        }).to(flowable -> Completable.merge(flowable, 100));
```

Comment:
What's this merge call for?

Reply:
Is this what limits the number of concurrent putAsync calls to a 100?

Reply:
Yes